bin/https.c

```diff
@@ -113,6 +113,10 @@ int https(struct s2n_connection *conn, uint32_t bench)
     BUFFER("Curve: %s\n", s2n_connection_get_curve(conn));
     BUFFER("KEM: %s\n", s2n_connection_get_kem_name(conn));
     BUFFER("KEM Group: %s\n", s2n_connection_get_kem_group_name(conn));
+    const char *group_name = NULL;
+    if (s2n_connection_get_key_exchange_group_name(conn, &group_name)) {
```

Comment:
I think the error checking here is currently incorrect. This function returns 0 on success.
```suggestion
    if (s2n_connection_get_key_exchange_group_name(conn, &group_name) == 0) {
```